server/src/main/java/org/opensearch/common/util/ThrowableTranslateUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.ParameterizedMessage;
+import org.opensearch.common.CheckedRunnable;
+import org.opensearch.common.CheckedSupplier;
+import org.opensearch.index.store.remote.fc.FileCache;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+/**
+ * Utils to translate/cast throwable instances
+ *
+ * @opensearch.internal
+ */
+final public class ThrowableTranslateUtils {
```

Comment:
Please remove this class, enhance `ExceptionsHelper` if needed

Reply:
yep totally make sense, thanks for pointing to that class